NitroxServer-Subnautica/Serialization/ResourceAssetsParser.cs

```diff
@@ -58,17 +82,457 @@ public static ResourceAssets Parse()
                                 wei.localScale = new UnityEngine.Vector3(resourcesFile.reader.ReadSingle(), resourcesFile.reader.ReadSingle(), resourcesFile.reader.ReadSingle());
                                 resourceAssets.WorldEntitiesByClassId.Add(wei.classId, wei);
                             }
-                        }
+                            break;
+                        case "ReefbackSlotsData":
```

Comment:
Let's make some new class here.  `WorldEntityDataParser` and `ReefbackSlotsDataParser`.  Then we can have a Dictionary<String, MonoBehaviourParser>`